examples/widget/db/prisma/migrations/20211027133033_init/migration.sql

```diff
@@ -0,0 +1,7 @@
+-- CreateTable
+CREATE TABLE "User" (
+    "externalId" TEXT NOT NULL,
```

Comment:
🔧 I'd suggest we also add a traditional `id` that can be used as a FK to show the recommended practise. WDYT?